<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInstrumentManager</class>
 <widget class="QDialog" name="DialogInstrumentManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>811</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>添加到：</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>41</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>工程：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxProject">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>41</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>服务表：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxService">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonDoAdd">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>执行添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDoDelete">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>440</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>执行删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
